app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -86,36 +91,31 @@ If you have a teaching qualification from the following countries and regions, y
 * Ukraine
 * the USA
 
-In addition, if you have a subject specialism in languages, mathematics or science and a teaching qualification to teach to teach 11 to 16 year olds from the following countries, you will be able to use the service to check if you meet the requirements for QTS: 
+In addition, if you have a subject specialism in languages, mathematics or science and a teaching qualification to teach to teach 11 to 16 year olds from the following countries, you will be able to use the service to check if you meet the requirements for QTS and apply: 
```

Comment:
maths

Reply:
leaving mathematics so it's the same across GOV.UK and here. And I think not re comma

Reply:
Not sure why but on GOV.UK they use mathematics on the guidance pages but it says maths in some of the other pages https://www.gov.uk/government/collections/national-curriculum?

I was just thinking for consistency as we say maths on the rest of the GIT site and use maths on our maths subject page https://getintoteaching.education.gov.uk/subjects/maths.

Reply:
No probs I will change